### DISCUSSION TITLE
日期可以任意点选多选。

### DISCUSSION THREAD

**Discussion by xiaqii** [layui/layui#1509] [问答]
### 议题条件

- [X] 我确认已查看官方使用文档：https://layui.dev ，但没有找到相关解决方案。
- [X] 我确认已在 [Issues](https://github.com/layui/layui/issues?q=is%3Aissue ) 中搜索过类似的问题，但没有找到相关解决方案。
- [X] 我已仔细阅读： 🍀[ Layui Issue 贡献指南](https://github.com/layui/layui/blob/main/CONTRIBUTING.md)

### 议题类型

功能请求

### 使用版本

2.9.2

### 问题描述

比如:要选中12月15日,12月17日,12月21日,12月25日。这4天。
场景是：在后台选择活动有效期。

### 业务代码

```auto
如有打扰抱歉了。。
```

### 浏览器

edge

### 友好承诺

- [X] 我承诺将本着相互尊重、理解和友善的态度进行交流，共同维护 Layui 良好的社区氛围。

**Comment by sentsim** [comment]
尝试借助 `autoConfirm` 选项结合相关的回调函数来自助实现。
相关文档：https://layui.dev/docs/2/laydate/#options
